docs/api/adbc.md

```diff
@@ -80,9 +80,15 @@ Functions related to binding, used for bulk insertion or in prepared statements.
 |:---|:---|:---|:---|
 | `StatementBindStream` |  Bind Arrow Stream. This can be used for bulk inserts or prepared statements.| `(AdbcStatement*, ArrowArrayStream*, AdbcError*)` | `StatementBindStream(&adbc_statement, &input_data, &adbc_error)` |
 
-## C++ Example
+## Examples
 
-We begin our example by declaring the essential variables for querying data through ADBC. These variables include Error, Database, Connection, Statement handling, and an Arrow Stream to transfer data between DuckDB and the application.
+Regardless of the programming language being used, there are two database
```

Comment:
please drop superfluous newlines after "two database" and "how you can do so".

Reply:
done